src/pybind/mgr/dashboard/frontend/src/styles.scss

```diff
@@ -1,2 +1,256 @@
 /* You can add global styles to this file, and also import other style files */
-@import './openattic-theme.scss';
+
+@import 'defaults';
+$fa-font-path: "../node_modules/fork-awesome/fonts";
+@import "../node_modules/fork-awesome/scss/fork-awesome";
+@import 'src/app/core/navigation/navigation.module';
```

Comment:
I'd try to either move that CSS code back here or leave it as a component CSS one, but not both ways, as I find it might be confusing.  Try the suggestion by @alfonsomthd to change the view encapsulation.